courses/machine_learning/images/mnist_models.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>Prefer os.path.join(a, b, c) over "a/ " + "/b" + "/c" to avoid issues with missing/too many "/"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done. Good call!</p>